Formula/a/arrayfire.rb

```diff
@@ -42,11 +47,18 @@ def install
     system "cmake", "-S", ".", "-B", "build",
                     "-DAF_BUILD_CUDA=OFF",
                     "-DAF_COMPUTE_LIBRARY=FFTW/LAPACK/BLAS",
+                    "-DCMAKE_CXX_STANDARD=17",
                     "-DCMAKE_INSTALL_RPATH=#{rpaths.join(";")}",
                     *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
 
+    # Remove debug info. These files make patchelf fail.
+    rm_f [
+      lib/"libaf.debug",
+      lib/"libafcpu.debug",
+      lib/"libafopencl.debug",
+    ]
```

Comment:
The Linux failure was with these debug info files, which patchelf.rb could not handle. Now we should be good.